### ISSUE TITLE
File layout

### ISSUE THREAD

**Issue by bmolyneaux** [Scipioceaser/Godot-Camera-Shake#1]
When you install this from the AssetLib, you are prompted which files to include because they conflict with the project files. Adding a camera_scripts folder directly to res:// is undesirable for most projects. The files should be organized like:
```
addons/camera_shake/shake_camera.gd
addons/camera_shake/shake_camera_2D.gd
LICENSE
README.md
```
so that they cleanly install into a project.

**Comment by bmolyneaux** [comment]
This isn't much of an issue, since it's easy enough to just select the camera script you need, and then move it in your project if desired. But I can send a PR to convert this to a plugin if you'd like.

Btw, I noticed that there is a demo folder, but it appears to be missing a scene file.

**Comment by awnagel** [comment]
Fixed with commit 890294f651e263fe5d44f7b6c4b59f11f6c1c0ba. Thank you for the input.
